onko_dicom.py

```diff
@@ -84,10 +85,19 @@ def open_file(self):
             logger.info("open_file user canceled open operation")
             return
 
-        self.plot_w.plot_dcm(full_path[0])
-        self.close_action.setEnabled(True)
+        try:
+            logger.info("Attempting to graph/open file")
```

Comment:
Should indicate what file.  If things go wrong, what can be done to investigate?
Where do you start?